### ISSUE TITLE
Layout XML vs Anko

### ISSUE THREAD

**Issue by stinodes** [Corsage/Masterani#3]
I created a similar project about a year ago, using Anko rather than XML for layouting, which ended up being great (albeit sometimes a little underdocumented, sadly), but could be fun to incorporate in the app as well to play around with 👍 

(not really criticism, rather just feedback/recommendation)

**Comment by Corsage** [comment]
I didn't hear about Anko until now. I only spent 10 min reading about it but so far it does seem super cool and interesting. I will probably make a revamped version later in the future, and would definitely consider using Anko. Thank you for the suggestion!

**Comment by stinodes** [comment]
No probs
I actually also made a similar app to learn some new fancy Android stuff (kotlin, anko, realm, Rx), though I didn't implement as much features but focused on the outlook and such instead
So your repo is a nice find for me 😀

**Comment by Corsage** [comment]
Yeah this is my project to learn Kotlin essentially. I have been learning RxJava/RxAndroid with Retrofit and am looking forward to incorporate it into this app in the future.

Design is something I need to work on, so Anko looks promising.

I'm glad to see that my repo is being of some use haha.

Thank you!
